circuit_knitting/cutting/qpd/decompose.py

```diff
@@ -198,6 +206,13 @@ def _decompose_qpd_instructions(
         data_id_offset += 1
         circuit.data.insert(i + data_id_offset, inst2)
 
+    # Get equivalence library
+    if translate_to_qpu is not None:
+        translate_to_qpu = translate_to_qpu.lower()
+    else:
+        translate_to_qpu = "standard"
```

Comment:
The `defaultdict` implementation of `equivalence_libraries` would support the `None` and behave correctly. I added this to appease mypy